src/apps/api/views/competitions.py

```diff
@@ -169,6 +174,19 @@ def email_all_participants(self, request, pk):
         batch_send_email.apply_async((comp.pk, content))
         return Response({}, status=status.HTTP_200_OK)
 
+    def _insure_organizer_participants_accepted(self, instance):
```

Comment:
should be `ensure`

Reply:
Oof, I was right the first time I wrote it then, ty

✔️